packages/styles/src/badges/test/badges.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Badges</title>
+  <link href="https://fonts.googleapis.com/css?family=Overpass+Mono|Roboto:300,400,500" rel="stylesheet">
+  <link rel="stylesheet" href="../../../../icons/dist/icons.css">
+  <link rel="stylesheet" href="../../../dist/airship.css">
+
+  <style>
+    html,
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    section {
+      display: flex;
+      flex-direction: column;
+      align-items: start;
+    }
+  </style>
+</head>
+
+<body>
+  <section>
+    <span class="as-badge as-badge--0"> Basic badge</span>
+    <span class="as-badge as-badge--1"> Basic badge</span>
+    <span class="as-badge as-badge--2"> Basic badge</span>
+    <span class="as-badge as-badge--3"> Basic badge</span>
+    <span class="as-badge as-badge--4"> Basic badge</span>
+    <span class="as-badge as-badge--5"> Basic badge</span>
+
+    <span class="as-badge as-badge--2"> Basic badge <i class="as-icon-close"></i></span>
```

Comment:
@IagoLast I'm getting here a 404 on icons:

![image](https://user-images.githubusercontent.com/458196/43890582-7465bc26-9bc7-11e8-997b-01e530c30c05.png)
Do I have to prepare the icons previously somehow?

On the other side, how about changing the descriptions to be more specific?
* Badge 0
* Badge 1
* ...
* Badge with icon

Reply:
ummm, I guess you should generate the icons manually I'll change it to get the icons from a CDN